docs/basics/manual-installation.md

```diff
@@ -29,6 +29,19 @@ txindex=1
 prune=0
 mempoolfullrbf=1
 rpcworkqueue=100
+zmqpubrawtx=tcp://0.0.0.0:9332
```

Comment:
okay do we have to update Docker Compose and Terraform as well?

Reply:
yep! done for docker compose!